packages/polaris-viz/src/components/DonutChart/Chart.tsx

```diff
@@ -208,60 +229,59 @@ export function Chart({
               viewBox={`${minX} ${minY} ${viewBoxDimensions.width} ${viewBoxDimensions.height}`}
               height={diameter}
               width={diameter}
+              ref={setSvgRef}
             >
-              {isLegendMounted && (
```

Comment:
I don't think the `isLegendMounted` check is used anymore and it was causing issues with attaching focus events to the arcs.